### PR TITLE
June 2019 newsletter

### PR DESCRIPTION
**Pull request by RangerMauve**
Here's the initial draft of the June 2019 newsletter. I also cleaned up the template a little.

Lemme know if I should change anything.

cc @dwblair

### CODE REVIEW

**Comment by dwblair** [comment]
Awesome! Will review by early eve if that’s not too late, apologies in
transit on small boat right now :)
Cheers!

File: 2019/2019-07.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+> Dat Project Newsletter Template
+
+![Dat Project logo](https://datproject.github.io/design/downloads/dat-data-logo.svg)
```
Comment (joehand) [inline 2019/2019-07.md]: Please use new logo =)

File: newsletter-template.md [hunk 2]
```diff
@@ -1,33 +1,14 @@
 > Dat Project Newsletter Template
 
-![Dat Project logo](https://datproject.org/public/img/dat-logo.png)
+![Dat Project logo](https://datproject.github.io/design/downloads/dat-data-logo.svg)
```
Comment (joehand) [inline newsletter-template.md]: How come we switched back to old logo?

File: 2019/2019-07.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+[Zach!](@ZqH7Mctu/7DNInxuwl12ECjfrAKUX2tBLq1rOldNhg0=.ed25519) and [Angelica](@eANNuLfzX/9rlGODXHYV8WJb+zw2h+d7YsT4vpYPvD0=.ed25519) from the [Secure Scuttlebutt](https://www.scuttlebutt.nz/) community recently put together [Song Islands](dat://songs.solarpunk.cool/) using Beaker. In their words "This is a Song Island. It contains a song, some words, and a picture." You can fork their arhive to create your own, or check out [their template](https://gitlab.com/dat-zines/song-island-template/)
+
+[READ MORE (On SSB)](%TYOYNAqUqpuNcGqHQcsAou8e/v7JqqG7wj3fG8xe9Fs=.sha256)
+
+## Cabal Updates
+
+The [Cabal](https://cabal.chat/) community has been busy this month with a bunch of new projects and updates.
+
+- [cblgh](https://github.com/cblgh) put toghether[cabal-headless](https://github.com/cabal-club/cabal-headless) to make it easier to build experiments on top of cabal
```
Comment (joehand) [inline 2019/2019-07.md]: space missing before `[cabal-headless]`

File: newsletter-template.md (hunk 2 above)
Comment (RangerMauve) [reply newsletter-template.md]: I thought that was the new logo! 😂

**Comment by RangerMauve** [comment]
Thank you for your reviews! I'll go put it up on Mailchip. :)

**Pull request merged**